data-access/nexustiles/dao/CassandraProxy.py

```diff
@@ -168,6 +168,7 @@ def __init__(self, config):
     def __open(self):
 
         dc_policy = DCAwareRoundRobinPolicy(self.__cass_local_DC)
+        #dc_policy = WhiteListRoundRobinPolicy([self.__cass_url ])
```

Comment:
Could you delete this commented line and the unused import in this file?

Reply:
Done